cloudevents/abstract/json_methods.py

```diff
@@ -0,0 +1,54 @@
+#  Copyright 2018-Present The CloudEvents Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import typing
+
+from cloudevents.abstract import AnyCloudEvent
+from cloudevents.sdk import types
+from cloudevents.abstract.http_methods import to_structured, from_http
+
+
+def to_json(
```

Comment:
I'd say it worth moving these `http_methods` and `json_methods` to the `cloudevents` package level. These have nothing "abstract` in them. They don't belong here.

Reply:
Isn't it confusing? 

what is the difference between `cloudevents.http.http_methods` and `cloudevents.http_methods`?

Reply:
Well, there's somewhat confusing APIs all over the package. But these are definitely not abstract. They look more like generic or standard implementations.

Reply:
Maybe even name the file with these APIs as `conversion.py`. This will then make a bit more sense.

Reply:
Maybe we should rename the whole module to `generic`?

Reply:
Generic is a bad name for a module IMO